src/reducers.js

```diff
@@ -14,8 +14,9 @@ const createRootReducer = history =>
     // The authentication state is added as initialState when
     // creating the store in data/store.js.
     authentication: identityReducer,
+    configuration: identityReducer,
```

Comment:
The configuration data was added to redux so connected components have access to it without knowing how it got there.